tests/unit/test_project_templates.py

```
@@ -0,0 +1,205 @@
+        project: Project,
+        template_data: CreateWorkItemTemplate,
+    ):
+        """Create a test work item template and yield it, then delete it."""
+        tmpl = client.project_templates.work_item_templates.create(
+            workspace_slug, project.id, template_data
+        )
+        yield tmpl
+        try:
+            client.project_templates.work_item_templates.delete(workspace_slug, project.id, tmpl.id)
+        except Exception:
+            pass
```

Ruff (0.15.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)